### DISCUSSION TITLE
label: how do I apply one?

### DISCUSSION THREAD

**Discussion by robert-hh** [micropython/micropython#9123] [General]
messages can be filtered by labels, and there are quite a few predefined, like port-stm32, port-esp32, port-rp2. but how di I apply a label to a message?

**Comment by robert-hh** [comment]
Ok. So I found it in the web view in the right column. There is a label field, which gets visible once you open the thread.

**Reply by jimmo** [reply]
Yeah, it would be nice to do this on the thread list view, rather than having to open the thread. I wonder if the GH app has a way to do this?

Open to suggestions on how to improve the workflow with labels etc. It was a bit surprising to me that the same labels are inherited from issues/prs, but I think that's workable.

Like I said on the forum, labelling threads isn't a use case that I've thought much about because I just read everything, but I agree it's useful if you're searching for something to know at-a-glance if a topic is likely to be relevant.

**Reply by robert-hh** [reply]
In the GH App the thread opens in the right column if you highlight it in the middle column. Then there is a little 'i' tagged button in the lower right corner which opens a window for tagging.
